File: ejercicios/c++/ejercicio071.cpp
#include <iostream>
using namespace std;

int a_binario_recursivo(int numero){
	if(numero == 0)
		return 0;
	else
		return (numero % 2) + (a_binario_recursivo(numero/2) * 10);
}


int main() {
	int numero;
	cout << "ingrese un numero entero : ";
	cin >> numero;
	
	if (numero >= 0)
		cout << a_binario_recursivo(numero);
	else
		cout << "Error el numero debe ser un entero positivo." << endl;
	return 0;
}

File: ejercicios/python/ejercicio071.py
def a_binario_recursivo(numero):
   if (numero == 0):
      return 0
   else:
      return (numero % 2) + (a_binario_recursivo(int(numero / 2)) * 10)

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   
   if (numero > 0):
      print(str(numero) + " en binario es : " + str(a_binario_recursivo(numero)))
   else:
      print("Error el numero debe ser un entero positivo.")